src/node.rs

```diff
@@ -42,11 +42,19 @@ impl Node {
     /// Returns `String` if `name` is `Some`
```

Comment:
The doc comment should cover the block case, maybe something like ".. and not `NodeName::Block`"